GeneratorInterface/RivetInterface/interface/RivetAnalysis.h

```diff
@@ -0,0 +1,169 @@
+#ifndef GeneratorInterface_RivetInterface_RivetAnalysis_H
+#define GeneratorInterface_RivetInterface_RivetAnalysis_H
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "Rivet/Analysis.hh"
+#include "Rivet/Projections/FinalState.hh"
+#include "Rivet/Particle.hh"
+#include "Rivet/Particle.fhh"
+#include "Rivet/Event.hh"
+#include "Rivet/Projections/FastJets.hh"
+#include "Rivet/Projections/JetAlg.hh"
+#include "Rivet/Projections/ChargedLeptons.hh"
+#include "Rivet/Projections/PromptFinalState.hh"
+#include "Rivet/Projections/DressedLeptons.hh"
+#include "Rivet/Projections/VetoedFinalState.hh"
+#include "Rivet/Projections/IdentifiedFinalState.hh"
+#include "Rivet/Projections/MissingMomentum.hh"
+#include "Rivet/Tools/RivetHepMC.hh"
+
+namespace Rivet {
+
+  class RivetAnalysis : public Analysis {
+
+    public:
+      vector<DressedLepton> leptons() const {return _leptons;}
```

Comment:
Hi @intrepid42 - please use the namespaces (even if presumably some rivet header incorrectly has a using namespace std

otherwise, could this be a const & instead?